bindings/c/test/apitester/TesterBlobGranuleCorrectnessWorkload.cpp

```diff
@@ -65,7 +67,10 @@ class ApiBlobGranuleCorrectnessWorkload : public ApiWorkload {
 
 	void debugOp(std::string opName, fdb::KeyRange keyRange, std::optional<int> tenantId, std::string message) {
 		if (BG_API_DEBUG_VERBOSE) {
-			info(fmt::format("{0}: [{1} - {2}) {3}: {4}",
+			info(fmt::format("{0}) {1}: [{2} - {3}) {4}: {5}",
+			                 std::chrono::duration_cast<std::chrono::duration<double>>(
```

Comment:
nit - create a local variable if it takes a long line to get current epoc? 
btw - does now() give what you need for epoch?

Reply:
This is outside of flow so there is no now() unfortunately